thumbor_shortener/handlers/shortener.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from thumbor_shortener.shortener import Shortener
+from thumbor.handlers.imaging import ImagingHandler
+from thumbor.url import Url
+
+class UrlShortenerHandler(ImagingHandler):
+
+    def get(self, **kwargs):
+
+        # Get the url from the shortener and parse the values.
+        url = Shortener.get(self.request.path)
```

Comment:
Yep

Thumbor's Context is awesome for that kind of stuff. The major hurdle will be to modify it to allow dynamic factories:

https://github.com/thumbor/thumbor/blob/master/thumbor/handlers/__init__.py#L432
https://github.com/thumbor/thumbor/blob/master/thumbor/server.py#L76